src/main/scala/uk/gov/hmrc/buildjobs/SbtBuildJobsPlugin.scala

```diff
@@ -56,10 +56,20 @@ object SbtBuildJobsPlugin extends sbt.AutoPlugin {
     writeVersion          := writeSettingValue(version         , EnvKeys.versionFilename         ).value,
     writeIsPublicArtefact := writeSettingValue(isPublicArtefact, EnvKeys.isPublicArtefactFilename).value,
     writeMeta             := writeMetaFile().value,
-    writeProjects         := writeTaskOutput(buildStructure.map(_.allProjects.map(r =>
-                               s"""|- module: ${r.id}
-                                   |  folder: ${r.base.getName}""".stripMargin
-                               ).sorted.mkString("\n")),
+    writeProjects         := writeTaskOutput(
+                               {
+                                 // pwd is the best way to identify the root module, since things like baseDirectory will change per module execution
+                                 val pwd = {import scala.sys.process._; "pwd" !!}.trim
```

Comment:
does java.nio Paths.get("") or sys props user.dir work for this? 
i guess it doesn't matter since it all does the same thing and we're not passing any args to it so no secrisk etc